static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -266,17 +267,17 @@ export class SentryAppExternalForm extends Component<Props, State> {
   renderField = (field: FieldFromSchema, required: boolean) => {
     // This function converts the field we get from the backend into
     // the field we need to pass down
-    let fieldToPass: Field = {
+    let fieldToPass: any = {
```

Comment:
Why is this changing to `any`? Like when would it not be a `Field` type?

Reply:
My bad, so while I was writing this PR, I had changed that to prevent having to deal with typing while I tested, it's been updated.